amical/data_processing.py

```diff
@@ -117,13 +117,22 @@ def select_data(cube, clip_fact=0.5, clip=False, verbose=True, display=True):
         ind_clip = []
         cube_cleaned_checked = np.array(good_fram)
 
+    cube_cleaned_checked = np.array(cube_cleaned_checked)
```

Comment:
`np.array` unconditionally copies data, even if the input variable is already an array. Since the input and output variable are the same, it's clear you don't *mean* to copy, so `np.asarray` is better suited here

```suggestion
    cube_cleaned_checked = np.asarray(cube_cleaned_checked)
```